src/_guides/language/sound-problems.md

```diff
@@ -358,6 +358,60 @@ in [Effective Dart](/guides/language/effective-dart/).
 
 <hr>
 
+<a name="uses-dynamic-as-bottom"></a>
+### A function of type ... can't be assigned
```

Comment:
Is there a better way to name this section? We usually summarize the issue for the TOC/heading, since the error string is already in the text.

Reply:
I don't have a great way to "explain" this unfortunately :-/